Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py

```diff
@@ -277,13 +313,13 @@ def get_content_records_context(content_records):
     return content_records_context
 
 
-def get_all_content_type_records(client, content_type, start_time, end_time):
-    content_blobs = client.list_content_request(content_type, start_time, end_time)
+def get_all_content_type_records(client, content_type, start_time, end_time, timeout=10):
+    content_blobs = client.list_content_request(content_type, start_time, end_time, timeout)
     # The list_content request returns a list of content records, each containing a url that holds the actual data
     content_uris = [content_blob.get('contentUri') for content_blob in content_blobs]
-    content_records = []
+    content_records: List = list()
```

Comment:
why did you need to change that?
`[]` is better than `list()`

Reply:
one of the lints raised an error and I remembered it once said list() is better. Why is [] better? :)

Reply:
https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better Found an answer